Model/Order/PlaceOrder.php

```diff
@@ -136,6 +136,8 @@ public function __construct(
         $this->checkoutSession = $checkoutSession;
     }
 
+    private const COMPLETE_ORDER_URL = '/checkout_sidekick/{{shopId}}/order/%s/state';
```

Comment:
Nitpick: Constants should be grouped together at the top of the class, before properties.

Reply:
Done here: e49bc3e2fe84ef7b965545b6232450fd79cf4329